bazel/BUILD

```diff
@@ -253,11 +253,6 @@ config_setting(
     values = {"define": "wasm=enabled"},
 )
```

Comment:
I don't think we need `wasm=enabled` either, since V8 is enabled by default.

Reply:
Done.